pkg/controller/servicebindingrequest/reconciler_test.go

```diff
@@ -399,3 +399,121 @@ func TestEmptyApplicationSelector(t *testing.T) {
 	require.Equal(t, corev1.ConditionTrue, sbrOutput.Status.Conditions[0].Status)
 	require.Equal(t, 0, len(sbrOutput.Status.Applications))
 }
+
+func TestBindTwoSbrsWithSingleApplication(t *testing.T) {
+	applicationResourceRef := "applicationResourceRef"
+	matchLabels := map[string]string{
```

Comment:
No point in using both `applicationResourceRef` as well as `matchLabels`.

Reply:
Yeah, I agree, I just kept it to be consistent with other tests.

Reply:
MatchLabels removed.